src/test/plugin/Plugins_test.cpp

```diff
@@ -605,6 +605,169 @@ class Plugins_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testTransactorPrivateFailure()
+    {
+        testcase("Transactor invalid private methods");
+
+        using namespace jtx;
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const carol{"carol"};
+        Account const danny{"danny"};
+
+        std::string const amendmentName = "featurePluginTest";
+        auto const trustSet2Amendment =
+            sha512Half(Slice(amendmentName.data(), amendmentName.size()));
+
+        cleanup();
+        PluginEnv env{
+            *this,
+            makeConfig("plugin_test_trustset.xrplugin"),
+            FeatureBitset{supported_amendments_plugins()},
+            trustSet2Amendment};
+
+        env.fund(XRP(5000), alice, bob, carol, danny);
+        IOU const USD = bob["USD"];
+        // sanity checks
+        BEAST_EXPECT(env.balance(alice) == XRP(5000));
+        BEAST_EXPECT(env.balance(bob) == XRP(5000));
+
+        // fakePayFee
+        {
+            Json::Value jv;
+            jv[jss::TransactionType] = "TrustSet3";
+            jv[jss::Account] = alice.human();
+            {
+                auto& ja = jv[jss::LimitAmount] =
+                    USD(1000).value().getJson(JsonOptions::none);
+                ja[jss::issuer] = bob.human();
+            }
+            auto const& tx = env.jt(jv);
+
+            Serializer s;
+            tx.stx->add(s);
+            auto const& jr = env.rpc("submit", strHex(s.slice()));
```

Comment:
You can submit the transaction with just `env(jv);` instead of needing this.

Reply:
I need to check response, and I didn't found other way to get it. Is it possible?

Reply:
You can do that with e.g. `env(jv, ter(tecINTERNAL));`